app/Models/Forum/TopicPoll.php

```diff
@@ -107,6 +111,7 @@ public function save()
                 'poll_length' => ($this->params['length_days'] ?? 0) * 3600 * 24,
                 'poll_max_options' => $this->params['max_options'],
                 'poll_vote_change' => $this->params['vote_change'] ?? false,
+                'poll_hide_results' => $this->params['hide_results'] ?? false,
```

Comment:
Defaults can be set in `fill` method.

<sub>And then no one remembered about that until now hence other entries also setting the defaults here.</sub>